algorithm/marine/soca_setcorscales.yaml.j2

```diff
@@ -7,6 +7,12 @@ date: "0001-01-01T00:00:00Z"
 
 corr variables: [sea_surface_height_above_geoid]
 
+islands:
+  # Palmico Sound
```

Comment:
Nit-picky, but this is a typo. "Pamlico Sound"

Reply:
dyslexia in action. 100% sure I'll wonder again how to spell this tomorrow am.